content/monitors/monitor_types/anomaly.md

```diff
@@ -155,16 +155,6 @@ The second graph shows the same metric, a day later. Even though it uses the pre
 
 {{< img src="monitors/monitor_types/anomaly/no_effect.png" alt="no effect" responsive="true" popup="true" >}}
 
-### Does past anomalies affect the current predictions?
```

Comment:
Can you change that to "Do past anomalies" in the one you keep?

Reply:
👍